includes/blocks/safe-svg/edit.js

```diff
@@ -155,47 +157,31 @@ const SafeSvgBlockEdit = ( props ) => {
 							onError={onError} />
 					</BlockControls></>
 			}
-			<MediaUpload
-				onSelect={onSelectImage}
-				allowedTypes={ALLOWED_MEDIA_TYPES}
-				accept={ALLOWED_MEDIA_TYPES}
-				value={imageID}
-				render={({open}) => {
-					return (
-						<div
-							style={{
-								maxWidth: '100%',
-								textAlign: alignment
-							}}
-						>
-							{!svgURL &&
-								<Button variant="tertiary" onClick={open}>
-									{__('Select an SVG icon', 'safe-svg')}
-								</Button>
-							}
-							{svgURL &&
-								<svg
-									style={{
-										width: dimensionWidth,
-										height: dimensionHeight,
-										maxWidth: '100%',
-										maxHeight: '100%'
-									}}
-								>
-									<image
-										xlinkHref={svgURL}
-										src={svgURL}
-										width={dimensionWidth < dimensionHeight ? dimensionWidth : '100%'}
-										style={{
-											height: dimensionWidth > dimensionHeight ? dimensionHeight : 'auto'
-										}}
-									/>
-								</svg>
-							}
-						</div>
-					);
-				}}
-			/>
+
+
+			{!svgURL &&
+				<MediaPlaceholder
+					onSelect={onSelectImage}
+					allowedTypes = {ALLOWED_MEDIA_TYPES}
+					accept={ALLOWED_MEDIA_TYPES}
+					value={imageID}
+					labels={{
+						title: __( 'Inline SVG', 'safe-svg' ),
+						instructions: __( 'Upload an SVG or pick one from your media library.', 'safe-svg' )
+					}}
+				/>
+			}
+
+			{svgURL &&
+				<div { ...containerBlockProps } className={className.replace(/(has-vivid-red-background-color|has-secondary-color|has-text-color|has-background)/g, '').trim()}>
```

Comment:
Maybe replace other background colors as well?
```suggestion
				<div { ...containerBlockProps } className={className.replace(/(has-vivid-.*-background-color|has-secondary-color|has-text-color|has-background)/g, '').trim()}>
```

Reply:
@iamdharmesh Yeah, great catch! Updated the regex so it removes all classes that start with `has-` and ends with `-color` along with the existing `has-background` class.